src/cli/util.js

```diff
@@ -269,7 +269,21 @@ function formatStdin(argv) {
 }
 
 function createIgnorer(argv) {
-  const ignoreFilePath = path.resolve(argv["ignore-path"]);
+  let ignorePaths = argv["ignore-path"];
+  if (!Array.isArray(ignorePaths)) {
+    ignorePaths = [ignorePaths];
+  }
+
+  const childIgnores = ignorePaths.map(createChildIgnore);
+
+  return childIgnores.reduce(
+    (parentIgnorer, childIgnorer) => parentIgnorer.add(childIgnorer),
+    ignore()
+  );
+}
+
+function createChildIgnore(ignorePath) {
```

Comment:
Does this work? Rename this function to `readIgnoreFile` and simply `return ignoreText`

Would make it more straight forward I believe.. but if it doesn't work it's fine =)

Reply:
When I read `ignore` library's README, I was under the impression it would not work but I did not try, following what they were saying about combining multiple `ignorer` together.

Let me try, I will report back.

Reply:
Yeah it works, modified code to use it now and renamed `ignoreText` to `ignoreFileContent` as I felt it was a better name, don't hesitate to tell if you want me to revert that change.